### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

**- What I did**
- Updated docs README. 

**- How to verify it**

**- Description for the changelog** README update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,25 @@
 
 You will find here **[Horusec's documentation](https://horusec.io/)**.
```
Comment (mdcpmoreira) [inline README.md:3]:
```suggestion
You will find here all about **[Horusec's documentation](https://horusec.io/)**.
```

File: README.md [hunk 2]
```diff
@@ -2,13 +2,25 @@
...
+> #### i. [**Requirements**](#requirements)
+> #### ii. [**Installation**](#installation)
+> #### iii. [**Build**](#build)
+3. [**How to contribute?**](#how-to-contribute)
+4. [**Roadmap**](#roadmap)
+5. [**Code of Conduct**](#code-of-conduct)
+6. [**Community**](#community)
+
+## **About**
 Horusec is an open-source tool that performs a static code analysis to identify security flaws during the development process. Currently, the languages for analysis are C#, Java, Kotlin, Python, Ruby, Golang, Terraform, Javascript, Typescript, Kubernetes, PHP, C, HTML, JSON, Dart, Elixir, Shell, Nginx.
```
Comment (mdcpmoreira) [inline README.md:17]:
```suggestion
Horusec is an open source tool that performs a static code analysis to identify security flaws during the development process. Currently, the languages for analysis are C#, Java, Kotlin, Python, Ruby, Golang, Terraform, Javascript, Typescript, Kubernetes, PHP, C, HTML, JSON, Dart, Elixir, Shell, Nginx. 
```

File: README.md [hunk 3]
```diff
@@ -2,13 +2,25 @@
...
+> #### ii. [**Installation**](#installation)
+> #### iii. [**Build**](#build)
+3. [**How to contribute?**](#how-to-contribute)
+4. [**Roadmap**](#roadmap)
+5. [**Code of Conduct**](#code-of-conduct)
+6. [**Community**](#community)
+
+## **About**
 Horusec is an open-source tool that performs a static code analysis to identify security flaws during the development process. Currently, the languages for analysis are C#, Java, Kotlin, Python, Ruby, Golang, Terraform, Javascript, Typescript, Kubernetes, PHP, C, HTML, JSON, Dart, Elixir, Shell, Nginx. 
 The tool has options to search for key leaks and security flaws in all your project's files, as well as in Git history. Horus can be used by the developer through the CLI and by the DevSecOps team on CI /CD mats.
```
Comment (mdcpmoreira) [inline README.md:18]:
```suggestion
The tool has options to search for key leaks and security flaws in all your project's files, as well as in Git history. Horusec can be used by the developer through the CLI and by the DevSecOps team on CI /CD mats.
```

File: README.md [hunk 4]
```diff
@@ -44,22 +54,25 @@ make run
 
 - Now, check if the `Web Server is available at http://localhost:1313/docs/`.
 
-## **Running the website locally**
+### **Running the website locally**
 
 Once you've cloned this repo, from the `root` folder, run:
```
Comment (mdcpmoreira) [inline README.md:59]:
```suggestion
Once you've cloned this repo, from the `root` folder, run the command:
```

File: README.md [hunk 5]
```diff
@@ -44,22 +54,25 @@ make run
...
 
 ```
 make build
 ```
 
-# **How to contribute?**
+## **How to contribute?**
 
 You can suggest a change, a fix, or improvements to our documentation, for example:
```
Comment (mdcpmoreira) [inline README.md:67]:
```suggestion
You can suggest a change, a fix, or improvements to our documentation, such as: 
```

**Review by mdcpmoreira — COMMENTED** [review]
TW review done ;)

File: README.md (hunk 1 above)
Comment (mdcpmoreira) [reply README.md:3]: I put this intro in the same pattern of Ritchie's docs repo

**Review by mdcpmoreira — COMMENTED** [review]
One more adjustment :)

**Comment by mdcpmoreira** [comment]
@wiliansilvazup can you review this PR, please?

**Comment by mdcpmoreira** [comment]
@wiliansilvazup @nathanmartinszup @lucasbrunozup @matheusalcantarazup @iancardosozup can someone merge this PR for us, please? :)

**Review by wiliansilvazup — APPROVED** [review]

**Pull request merged**